extensions/binding/org.eclipse.smarthome.binding.ble/src/main/java/org/eclipse/smarthome/binding/ble/BleBridgeApi.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.binding.ble;
+
+/**
+ * The {@link BleBridgeApi} class defines the standard bridge API that must be implemented by Thing implemented in the
+ * BLE host.
+ * <p>
+ * <b>Scanning</b>
+ * The API assumes that the bridge is "always" scanning to enable beacons to be received.
+ * The bridge must decide to enable and disable scanning as it needs. This design choice avoids interaction between
+ * higher layers where a binding may want to enable scanning while another needs to disable scanning for a specific
+ * function (e.g. to connect to a device). The bridge should disable scanning only for the period that is needed.
+ *
+ * @author Chris Jackson - Initial contribution
+ */
+public interface BleBridgeApi {
```

Comment:
Any reason why this should not be called `BleBridgeHandler` (and possibly also extend `BridgeHandler`? 
After all, all instances are `BridgeHandlers` and that's how you get hold of them. This way, it would also be clear that they respect the Thing lifecycle, so e.g. using the api after disposal of the handler is not a good idea.